app/src/main/java/com/brewthings/app/data/RaptPillParser.kt

```diff
@@ -0,0 +1,28 @@
+package com.brewthings.app.data
+
+import com.brewthings.app.data.model.RaptPillData
+import java.nio.ByteBuffer
+import java.nio.ByteOrder
+
+class RaptPillParser() {
+    fun parse(data: ByteArray): RaptPillData {
+        if (data.size!= 23) {
+            throw IllegalArgumentException("Metrics data must have length 23")
+        }
```

Comment:
25 oppure 21. non facciamo come il cretino che trimma `R` e `A` e lascia `P` e `T` 🤣 o lasciamo tutto o tagliamo tutto.

Reply:
Sisi, per ora ho solo fatto fare mezza traduzione a chat gpt, poi mi ricontrollo la logica con calma

Reply:
sisí ho visto che la PR è in draft. infatti non è una review, ti sto lasciando i commenti singoli come promemoria XD

Reply:
credo di aver capito perchè il cretino ha scritto il codice cosí XD in pratica, sta parsando il manufacturer data che, effettivamente, è lungo 23 bytes (quando è valido) e inizia con `PT`. Proviamo a lasciarlo cosí il codice, e io ti passo il manufacturer data.
<img width="204" alt="Screenshot 2024-05-10 at 22 45 17" src="https://github.com/SeTzener/Brewthings/assets/3224791/1e9baf9a-0172-46ce-bedf-4690cefdcba8">